File: go_by_tdd/tdd_hello_world/hello_world.go
package main

import "fmt"

func Hello(name string, lang string) string {
	prefix := "Hello, "
	if name == "" {
		name = "World"
	}
	if lang == "Spanish" {
		prefix = "Hola, "
	}
	return prefix + name
}

func main() {
	fmt.Println(Hello("Warren", ""))
}

File: rust/hello_world/hello_world.rs
fn main() {
    // Statements here are executed when the compiled binary is called

    // Print text to the console
    
    let name = "Warren";
    let age = 27;
    println!("{}" , "Hello World!");
    println!("{} {}" , "My Name is", name);
    println!("{} {} {}" , "I am" , age, "years old");
    println!("{} {} {}" , "I am turning", add_one(age), "on the 23rd of July");
}

fn add_one(k: i32) -> i32{
    return k+1;
}

